tests/functional/utils/MockServerHelper.cpp

```diff
@@ -66,6 +66,27 @@ void MockServerHelper::MockLookupPlatformApiResponse(
       std::move(data), "/lookup/v1/platform/apis");
 }
 
+void MockServerHelper::MockGetResponse(const std::string& layer,
+                                       const std::string& data_handle,
+                                       const std::string& data) {
+  mock_server_client_.MockResponse("GET",
+                                   "/blob/v1/catalogs/" + catalog_ +
+                                       "/layers/" + layer + "/data/" +
+                                       data_handle,
+                                   data);
+}
```

Comment:
could we change this method in future to void MockServerHelper::MockGetResponse(
                                       const std::string& data,  const std::string& path)